docs/aks/demo/DEMO_SCRIPT.md

```diff
@@ -0,0 +1,389 @@
+- Bash shell (tested on Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell unless you have a remote dockerd
+- Azure CLI 2.0.72+ ([download](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- Docker CLI ([download](https://docs.docker.com/install/))
+- .NET Core SDK 2.2 ([download](https://dotnet.microsoft.com/download))
+- Visual Studio Code (optional) ([download](https://code.visualstudio.com/download))
+- kubectl (install by using `sudo az aks install-cli`)
+- Helm v3 ([Install Instructions](https://helm.sh/docs/intro/install/))
+
+### Setup
+
+Fork this repo and clone to your local machine
+
+```shell
+cd $HOME
+
+mkdir demo
+
+cd demo
+
+git clone https://github.com/RetailDevCrews/helium
+```
+
+Change into the base directory of the repo
+
+```shell
+cd helium
+```
+
+Login to Azure and select subscription
+
+```bash
+
+az login
+
+# show your Azure accounts
+az account list -o table
+
+# select the Azure account
+az account set -s {subscription name or Id}
+
+```
+
+This demo will create resource groups, a CosmosDB instance, Key Vault, Azure Container Registry, and Azure App Service.
+
+Choose a unique name for DNS and resource name prefix.
+
+```bash
+
+# this will be the prefix for all resources
+# do not include punctuation - only use a-z and 0-9
+# must be at least 5 characters long
+# must start with a-z (only lowercase)
+export He_Name="youruniquename"
+
+### if true, change He_Name
+az cosmosdb check-name-exists -n ${He_Name}
+
+### if nslookup doesn't fail to resolve, change He_Name
+nslookup ${He_Name}.azurewebsites.net
+nslookup ${He_Name}.vault.azure.net
+nslookup ${He_Name}.azurecr.io
+
+```
+
+Create Resource Groups
+
+- When experimenting with this sample, you should create new resource groups to avoid accidentally deleting resources
+
+  - If you use an existing resource group, please make sure to apply resource locks to avoid accidentally deleting resources
+
+- You will create 3 resource groups
+  - One for CosmosDB
+  - One for ACR
+  - One for App Service, Key Vault and Azure Monitor
+
+```bash
+
+# set location
+export He_Location=centralus
+
+# resource group names
+export He_ACR_RG=${He_Name}-rg-acr
+export He_App_RG=${He_Name}-rg-app
+export He_Cosmos_RG=${He_Name}-rg-cosmos
+
+# create the resource groups
+az group create -n $He_App_RG -l $He_Location
+az group create -n $He_ACR_RG -l $He_Location
+az group create -n $He_Cosmos_RG -l $He_Location
+
+```
+
+Save your environment variables for ease of reuse and picking up where you left off.
+
+```bash
+
+# run the saveenv.sh script at any time to save He_* variables to ~/${He_Name}.env
+# make sure you are in the root of the repo
+./saveenv.sh
+
+# at any point if your terminal environment gets cleared, you can source the file
+# you only need to remember the name of the env file (or set the $He_Name variable again)
+source ~/{yoursameuniquename}.env
+
+```
+
+Create and load sample data into CosmosDB
```

Comment:
I was trying to make this a single demo script instead of having someone jump back and forth. Makes sense though to include a link to the csharp docs so that it can used and this keeps implementation for that linked there and if any implementation changes are needed for helium-csharp those changes don't have to be made multiple places.